hw/ip/sysrst_ctrl/data/sysrst_ctrl.hjson

```diff
@@ -507,6 +507,9 @@
       swaccess: "rw1c",
       hwaccess: "hwo",
       resval: "0",
+      tags: [ // the value of these regs is determined by the
```

Comment:
@esshiu just to double check.

Reply:
For the adc_ctrl and sysrst_ctrl, we combine all the interrupt sources into one signal to PLIC. The interrupt status CSR has the detailed information. The tag looks reasonable (Only the hardware can set it).